src/Elmish.WPF/Elmish.WPF.fsproj

```diff
@@ -29,6 +30,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <None Include="..\..\README.md" Link="README.md" Pack="True" PackagePath="/" />
```

Comment:
What does this line do?

Reply:
The combination of this line and the above line will inline the README into the package description at [Elmish.WPF's nuget.org page](https://www.nuget.org/packages/Elmish.WPF#readme-body-tab).

See this stackoverflow answer: https://stackoverflow.com/a/70919720/1369188